CMSgov/mdct-mcr
# Enter access measure details

## Description

Resolves https://qmacbis.atlassian.net/browse/MDCT-1982
Also fixes this bug: https://qmacbis.atlassian.net/browse/MDCT-2017

Now that access measures can be created and saved via the modal, we need to be able to add data in the drawer and save it to the database. This PR:
- Adds completed state for EntityCard, including changing status icon and displaying details on the card
- Adds details section to the top of the drawer
- Adds `onSubmit` handler to ModalDrawerReportPage
- Cleans up StandardReportPage and DrawerReportPage to match

### How to test

1. Go the access measures section
2. Create several access measures
3. Enter measure details using the drawer
4. See that details are saved to the correct measure in the database and displayed correctly in the UI.

### Changed Dependencies

n/a

## Code author checklist

## Reviewer checklist (two different people)
